pkgs/top-level/all-packages.nix

```diff
@@ -10717,6 +10717,10 @@ in
     inherit (darwin.apple_sdk.frameworks) CoreServices ApplicationServices;
   };
 
+  julia_16 = callPackage ../development/compilers/julia/1.6.nix {
+    inherit (darwin.apple_sdk.frameworks) CoreServices ApplicationServices;
```

Comment:
It seems like these inputs are no longer needed in the derivation? It looks like they've been commented out so they can prob be removed here as well.